common/config/rush/nonbrowser-approved-packages.json

```diff
@@ -242,6 +250,10 @@
       "name": "ajv",
       "allowedCategories": [ "libraries" ]
     },
+    {
+      "name": "anymatch",
```

Comment:
Let's discuss this.

Globbing seems like an important performance concern:

- the algorithm crawls folders recursively, which can be super expensive e.g. if it accidentally crawls into `node_modules`, or if the same tree is getting crawled multiple times
- there are lots of possibilities for optimizing pattern matchers -- some libraries are clearly much faster than others
- globs can also benefit from caching
- certain APIs like `glob` are awkward to use or not even promisified

Also, each library implements a slightly different glob syntax, which is important for example if we need to escape a glob.

Scanning the repo, Rush Stack projects are already using `fast-glob`, `glob`, and `minimatch`.  Before we pile `anymatch` on top of that, let's agree that it's really the Recommended Best Awesome solution that we should adopt, and then deprecate all the others.

Reply:
Replace this with minimatch for now. Let's put together a library function for this soon.